edgePy/DGEList.py

```diff
@@ -191,39 +245,6 @@ def library_size(self) -> np.array:
         """
         return np.sum(self.counts, 0)
 
-    @classmethod
-    def read_handle(cls, handle: StringIO, **kwargs: Mapping) -> "DGEList":
-        """Read in a file-like object of delimited data for instantiation.
-
-        Args:
-            handle: Any handle supporting text streaming io.
-            kwargs: Additional arguments supported by ``np.genfromtxt``.
-
-        Returns:
-            DGEList: Container for storing read counts for samples.
-
-        """
-        # First column is the header for the the gene names.
-        # Remaining columns are sample names.
-        _, *samples = next(handle).strip().split()
-
-        genes = []
-        frame = np.genfromtxt(
-            fname=handle,
-            dtype=np.int,
-            converters={0: lambda _: genes.append(_) or 0},  # type: ignore
-            autostrip=kwargs.pop("autostrip", True),
-            replace_space=kwargs.pop("replace_space", "_"),
-            case_sensitive=kwargs.pop("case_sensitive", True),
-            invalid_raise=kwargs.pop("invalid_raise", True),
-            **kwargs,
-        )
-
-        # Delete the first column as it is copied on assignment to `genes`.
-        counts = np.delete(frame, 0, axis=1)
-
-        return cls(counts=counts, samples=samples, genes=genes[1:])
-
     def cpm(self, log: bool = False, prior_count: float = PRIOR_COUNT) -> None:
```

Comment:
Shouldn't normalization methods return a new DGEList instance, or at least a numpy view so that users can utilize multiple normalization methods (ex. within-smaples vs between-samples etc.) on the same data without manually creating multiple identical DGELists?

Reply:
Lets create an issue for this.  It could be done that way.

Reply:
New issue created - https://github.com/r-bioinformatics/edgePy/issues/61